charybdis/tests/integrations/custom_fields.rs

```diff
@@ -0,0 +1,107 @@
+use charybdis::scylla::deserialize::DeserializeValue;
+use charybdis::scylla::frame::response::result::ColumnType;
+use charybdis::scylla::SerializeValue;
+use charybdis::types::Text;
+
+#[derive(Debug, Default, Clone, PartialEq, strum::FromRepr)]
+#[repr(i8)]
+pub enum AddressTypeCustomField {
+    #[default]
+    HomeAddress = 0,
+    WorkAddress = 1,
+}
+
+#[derive(Debug)]
+struct AddressTypeCustomDeserializeErr(i8);
+impl std::fmt::Display for AddressTypeCustomDeserializeErr {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "AddressTypeCustomDeserializeErr({})", self.0)
+    }
+}
+impl std::error::Error for AddressTypeCustomDeserializeErr {}
+
+impl<'frame, 'metadata> DeserializeValue<'frame, 'metadata> for AddressTypeCustomField {
+    fn type_check(
+        typ: &scylla::frame::response::result::ColumnType,
+    ) -> std::result::Result<(), scylla::deserialize::TypeCheckError> {
```

Comment:
here, path prefix for `ColumnType` and `Result` is not needed it's already in the scope because of `use charybdis::scylla::frame::response::result::ColumnType;` and because `std::result::Result` is automatically in the scope.

Reply:
Let's update this in other functions as well.